velox/core/QueryConfig.h

```diff
@@ -97,10 +97,15 @@ class QueryConfig {
   static constexpr const char* kMaxPartitionedOutputBufferSize =
       "driver.max-page-partitioning-buffer-size";
 
-  /// Preffered number of rows to be returned by operators from
+  /// Preferred size of batches in bytes to be returned by operators from
```

Comment:
Let's update the docs: https://facebookincubator.github.io/velox/configs.html

Reply:
Done